CONTRIBUTING.md

```diff
@@ -182,6 +182,92 @@ components/
     my-component.spec.js
 ```
 
+#### Spec File Setup
+
+Each component will require a DOM-like environment in order to be queried for the custom element and, thus assert on the component details.
+
+The following is an example of a spec file, with imports, a global mock (`fetch`), and the `describe`/`it` blocks.
+
```

Comment:
Still mentions the global mock here.

If the main goal is to show the minimum boilerplate for a spec file, then it should probably focus more on this block, no?
```js
before(async () => {
  header = document.createElement("app-header");
  document.body.appendChild(header);

  await header.updateComplete;
});
```

I know you go into more detail later, but just trying to align the code sample to introductory text is all.  Maybe just keep this one simple and just related to testing, not specifically components?  
```js
import { expect } from "@esm-bundle/chai";
import { add } from "./math.js";

describe("Add Function", () => {
  it("should do something expected", () => {
    expect(add(1,2)).equal(3);
  });
});
```

Reply:
Rephrased this a bit, and changed the example to match yours.